modules/registry-accessing-directly.adoc

```diff
@@ -31,19 +37,26 @@ endif::[]
 . Log in to the container image registry by using your access token:
 +
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
+[source,terminal]
 ----
 sh-4.2# oc login -u kubeadmin -p <password_from_install_log> https://api-int.<cluster_name>.<base_domain>:6443
+----
++
+[source,terminal]
+----
 sh-4.2# podman login -u kubeadmin -p $(oc whoami -t) image-registry.openshift-image-registry.svc:5000
 ----
 endif::[]
 ifdef::openshift-dedicated[]
+[source,terminal]
 ----
 $ podman login -u $(oc whoami) -p $(oc whoami -t) $(oc -n openshift-image-registry get route default-route -o jsonpath='{.spec.host}')
 ----
 endif::[]
-+
+
```

Comment:
@bobfuru, shouldn't this be a continuation of the `.` step on line 37?

Reply:
Yes, good catch! I've fixed it so that the "You should see a message..." is indented with the `.` step on line 37.